testplan/web_ui/testing/src/AssertionPane/AssertionTypes/TableAssertions/tableAssertionUtils.js

```diff
@@ -1,8 +1,119 @@
 /** @module tableAssertionUtils */
 
 import React from "react";
-import {domToString} from './../../../Common/utils';
+import { useParams, Link } from "react-router-dom";
+import _ from "lodash";
+import {
+  domToString,
+  generateURLWithParameters
+} from './../../../Common/utils';
+
+// Small component for rendering TableLogLinkCell.
+function LinkRender(props) {
+  const { uid } = useParams();
+  if (props.inner) {
+    const link = generateURLWithParameters(
+      window.location,
+      props.link.startsWith("/")
+        ? `/testplan/${uid}${props.link}`
+        : `/testplan/${uid}/${props.link}`
+    );
+    return <Link to={link}>{props.title || props.link}</Link>;
+  } else {
+    return (
+      <a
+        rel="noreferrer"
+        href={props.link}
+        target={props.new_window ? "_blank" : "_self"}
+      >
+        {props.title || props.link}
+      </a>
+    );
+  }
+}
+
 
+// The cell of link.
+class TableLogLinkCell {
+  constructor(cell) {
+    this.cell = cell;
+    this.value = cell.title || cell.link;
+  }
+
+  render() {
+    return <LinkRender {...this.cell}/>;
+  }
+
+  compare(linkB) {
+    if (linkB instanceof TableLogLinkCell) {
+      return this.value >= linkB.value ? 1 : -1;
```

Comment:
Maybe not a concern but comparison should be w 3 way thing, treating equal similar as greater might mean that ordering back and forth will result different order.